packages/checkout-button-integration/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "root": "packages/checkout-button-integration",
+  "sourceRoot": "packages/checkout-button-integration/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/checkout-button-integration/**/*.{ts,tsx}"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": [
+        "coverage/packages/checkout-button-integration"
+      ],
+      "options": {
+        "jestConfig": "packages/checkout-button-integration/jest.config.js"
+      }
+    }
+  },
+  "tags": [
+    "scope:integration",
```

Comment:
I didn't tag `payment-integration` with `scope:integration`. I assumed it was for a payment method not API. Do I need to add it to `payment-integration`?

Reply:
No I think it's fine. As you said, `scope:integration` is for packages, when loaded, would integrate a payment method into checkout. Whereas `payment-integration-api` is a package consumed by the aforementioned packages to make the integration happen.